### PR TITLE
[BEAM-362] Move a few more remnants of internal state to runners/core-java

### PR DESCRIPTION
**Pull request by kennknowles**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [x] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [x] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [x] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by kennknowles** [comment]
R: @staslev just spreading the burden of my boring move PRs. This is a no-op but I'm trying to get the SDK cleaned up whenever it overlaps with something I am really working on.

The tests fail because the Dataflow service needs a corresponding update, but I like to get the external PR mostly approved first before I do that, and then I can fix it up, get green, and merge.

File: runners/core-java/src/main/java/org/apache/beam/runners/core/StateTags.java [hunk 1]
```diff
@@ -43,6 +51,66 @@
...
+
+      @Override
+      public <T> BagState<T> bindBag(
+          String id, StateSpec<? super K, BagState<T>> spec, Coder<T> elemCoder) {
+        return binder.bindBag(tagForSpec(id, spec), elemCoder);
+      }
+
+      @Override
+      public <InputT, AccumT, OutputT>
+      AccumulatorCombiningState<InputT, AccumT, OutputT> bindCombiningValue(
```
Comment (staslev) [inline runners/core-java/src/main/java/org/apache/beam/runners/core/StateTags.java]: Missing indent?

Comment (kennknowles) [inline runners/core-java/src/main/java/org/apache/beam/runners/core/StateTags.java]: Yea, just copy/paste of prior code. Autoformatted now.

**Comment by kennknowles** [comment]
Run Dataflow RunnableOnService

**Comment by kennknowles** [comment]
Run Dataflow RunableOnService

**Comment by staslev** [comment]
LGTM.

**Pull request merged**